homeassistant/components/camera/__init__.py

```diff
@@ -168,6 +176,16 @@ class Image:
     return await camera.handle_async_mjpeg_stream(request)
 
 
+@bind_hass
+async def async_get_mjpeg_stream_url(hass, entity_id):
```

Comment:
This doesn't seem to be used ? Let's not add it because mjpeg should buuuuurrn (stream is so much better)

Reply:
Removed it for now. 
Alexa supports mjpeg as well; originally I had plans to support it too. I think it has it's place where bandwidth or processing power might be an issue.

Reply:
Some devices support mjpeg, and in that case it's the most efficient for us.

However, if not, we mimick mjpeg by requesting images from the device and sending those over, creating our own stream. This requires more processing power, and because mjpeg doesn't have keyframes or any type of video compression (it's just jpeg images), it is not bandwidth efficient. Well, except for thie shitty frame rate I guess 😆 

Our rtsp streams are in-memory repackages to HLS. They are not re-encoded, just the container format is changed.

Reply:
I do btw think that with some changes to the camera integration we could make it possible to create an RTSP passthrough or even link the raw stream directly to Alexa. That's for the future though :)

Reply:
Because of the domain, https, everything on port 443, and trusted cert requirements it would have to be a pass-through like stream. Otherwise I would have submitted a PR for passing along any local camera RTSP input streams months ago. This was the first capability I attempted to implement when I started contributing, but ran into too many hurdles. I would like to tackle a WebRTC implementation, which would create a whole world of uses.